test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -72,6 +72,86 @@ def test_server_error
     end
   end
 
+  def test_prometheus_successful_call
+    setup_gce_metadata_stubs
+    setup_prometheus
+    setup_logging_stubs(false, 0, 'OK') do
```

Comment:
I'm a little concerned about the code duplication here, but not enough to block this PR on that.